### PR TITLE
Add ActiveNode::Base.create method

### PR DESCRIPTION
**Pull request by gnapse**

### CODE REVIEW

File: lib/active_node/validations.rb [hunk 1]
```diff
@@ -34,6 +34,18 @@ module Validations
...
+        if attributes.is_a?(Array)
+          attributes.collect { |attr| create(attr, &block) }
+        else
+          object = new(attributes)
+          yield(object) if block_given?
+          object.save
+          object
+        end
+      end
+
```
Comment (klobuczek) [inline lib/active_node/validations.rb]: Is this how they have it implemented in ActiveRecord? In an non DRY way, repeating nearly identical blocks?

**Comment by klobuczek** [comment]
Need to figure out as well why the build is failing.

File: lib/active_node/validations.rb (hunk 1 above)
Comment (gnapse) [inline lib/active_node/validations.rb]: Basically yes ([see here](https://github.com/rails/rails/blob/master/activerecord/lib/active_record/persistence.rb#L29-L54)). I did not bother to check before, but I figured that in spite of the similarities, trying to make it in a DRYer manner, would have involved some pretty nasty and not-at-all-clear meta-programming playing with the names of the methods.

Comment (gnapse) [inline lib/active_node/validations.rb]: BTW, by checking in ActiveRecord's code, I realized this perhaps should not be in the validations module, but in the persistence module. It kinda makes sense. If you want I can move both methods over there as part of the PR.

Comment (klobuczek) [inline lib/active_node/validations.rb]: yes. That would be much better.

Comment (klobuczek) [inline lib/active_node/validations.rb]: And now you are eligible to receive free license of rubymine :)

Comment (gnapse) [inline lib/active_node/validations.rb]: Nice! What do I need to do to get it? BTW, I'll resume my work on this these days. I do have a couple more suggestions to make this more like ActiveRecord, and will include them as well.

**Pull request merged**